packages/firebase_auth/firebase_auth/test/firebase_auth_test.dart

```diff
@@ -230,6 +231,37 @@ void main() {
       });
     });
 
+    group('customAuthDomain', () {
+      test('set customAuthDomain should call delegate method', () async {
+        // Each test uses a unique FirebaseApp instance to avoid sharing state
+        final app = await Firebase.initializeApp(
+            name: 'customAuthDomainTest',
+            options: const FirebaseOptions(
+                apiKey: 'apiKey',
+                appId: 'appId',
+                messagingSenderId: 'messagingSenderId',
+                projectId: 'projectId'));
+
+        FirebaseAuthPlatform.instance =
+            FakeFirebaseAuthPlatform(customAuthDomain: 'foo');
+        auth = FirebaseAuth.instanceFor(app: app);
+
+        expect(auth.customAuthDomain, 'foo');
+        if (defaultTargetPlatform == TargetPlatform.windows || kIsWeb) {
+          try {
+            auth.customAuthDomain = 'bar';
```

Comment:
Why not testing the getter as well for windows

Reply:
As mentioned in chat, this is just to illustrate what happens for each platform. The getter will always be null as we throw exception on setter.